gallery/plot_visualization_utils.py

```diff
@@ -131,8 +121,8 @@ def show(imgs):
 model = fcn_resnet50(weights=weights, progress=False)
 model = model.eval()
 
-normalized_batch = transforms(batch)
```

Comment:
Previously we were doubly preprocessing the images. I've changed it to do it only once using the original data (dog_list).